src/applications/vaos/services/healthcare-service/index.js

```diff
@@ -82,6 +82,30 @@ export async function getSupportedHealthcareServicesAndLocations({
 
   return results.filter(item => item.resourceType === 'Location');
 }
+/**
+ * Fetches a single clinic based on the id provided from the v2 endpoints
+ *
+ * @export
+ * @param {Object} params
+ * @param {string} params.locationId The location or facility id of the clinic
+ * @param {string} params.id The clinic id
+ */
+export async function fetchHealthcareServiceById({ locationId, id }) {
+  try {
+    const results = await getClinics({
+      locationId,
+      clinicIds: [id],
+    });
+
+    return transformClinicsV2(results)[0];
```

Comment:
Is the `[0]` necessary? The v2 endpoint should only return a single clinic if `clinicIds` is passed a single id array.

Reply:
Disregard this. I misread as `results[0]` and thought we were passing the first clinic we got in the response. This is actually grabbing the only clinic in the transformed array. Looks good.